### PR TITLE
Update the default connection creation timeout to 10s from 2s

### PR DESCRIPTION
**Pull request by nizarm**
Update the default connection creation timeout to 10s from 2s
This will be made configurable in a subsequent release

### CODE REVIEW

File: r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java [hunk 1]
```diff
@@ -822,11 +822,11 @@ public void testCreationTimeout(int poolSize, int concurrency) throws Exception
 
       // run for the duration of default creation timeout
       // TODO: parameterize the creation duration when the default creation gets parameterized
-      clockedExecutor.runFor(2000);
+      clockedExecutor.runFor(10000);
```
Comment (ChaoLinkedIn) [inline r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java]: use DEFAULT_OBJECT_CREATION_TIMEOUT

File: r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java [hunk 2]
```diff
@@ -822,11 +822,11 @@ public void testCreationTimeout(int poolSize, int concurrency) throws Exception
...
     }
 
     // drain all the pending tasks
-    clockedExecutor.runFor(2000);
+    clockedExecutor.runFor(10000);
```
Comment (ChaoLinkedIn) [inline r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java]: ditto

**Comment by nizarm** [comment]
> You need to change it again next time when you change this back to 2000, right? How about define a constant and use it everywhere?

Thanks - Updated with common constant.

File: r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java (hunk 2 above)
Comment (nizarm) [reply r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java]: Fixed

File: r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java (hunk 1 above)
Comment (nizarm) [reply r2-core/src/test/java/test/r2/transport/http/client/TestAsyncPool.java]: Fixed

**Pull request merged**